### ISSUE TITLE
[Feature Request] placeholder

### ISSUE THREAD

**Issue by Hannah-Doerpholz** [nfdi4plants/ARCitect#69]
**Is your feature request related to a problem? Please describe.**
It is often hard to tell which assays are linked to which study (especially if they are linked to multiple studies).

**Describe the solution you'd like**
It would be great to have something like a graph overview that shows all studies and how they are linked to all assays. If possible, it would then also be great if users could click on one of the graph nodes to then be automatically lead to the respective page (of that assay or study).

**Comment by Brilator** [comment]
Discussed this with @HLWeil last week. 
This might be more applicable to DataHUB / ISA Model than (ARCitect alone). 

I would envision this to be automatically be updated upon commit in the Hub (just like the validation pipeline).

**Comment by HLWeil** [comment]
Yep, I agree that this feature will help users keep track of their ARCs. We will probably first try to implement some back-end functionality for this in ARCtrl (https://github.com/nfdi4plants/ARCtrl/issues/164).
